.github/workflows/mac-tests.yml

```
@@ -72,7 +72,7 @@ jobs:
         python-version: 3.8
 
     - name: Setup cache
-      uses: actions/cache@v2
+      uses: actions/cache@v3
```

See https://github.com/optuna/optuna/pull/4207#issuecomment-1357135697
```suggestion
      uses: actions/cache@v2
```